tests/test_testing.py

```diff
@@ -133,3 +134,15 @@ def test_etag_secondary(client: Client, app: Teal):
         db.session.commit()
     _, r = client.get('/', item='NFCID', accept=ANY, status=302)
     assert r.location == 'https://dh.com/tags/FO-3MP5M/device'
+
+
+@pytest.mark.mvp
+def test_get_version(app: Teal, client: Client):
+    """Checks GETting versions of service."""
+
+    content, res = client.get("/versions/version/", None)
+    with open("ereuse_tag/__init__.py", encoding="utf8") as f:
+        version = re.search(r'__version__ = "(.*?)"', f.read()).group(1)
+    tag_version = version
```

Comment:
Why there are two variables with the same value.?

Reply:
the test was cleaned with __version__ var